build.sbt

```diff
@@ -41,7 +41,7 @@ lazy val core = project
       // https://mvnrepository.com/artifact/org.typelevel/cats-effect
       "org.typelevel" %%% "cats-effect" % "3.5.7",
```

Comment:
Do we need to change our cats effect dependency to 3.6.0? The [otel4s 0.12.0 release](https://github.com/typelevel/otel4s/releases/tag/v0.12.0) notes say:

> This version relies on new functionality from Cats Effect [3.6.0](https://github.com/typelevel/cats-effect/releases/tag/v3.6.0), such as IORuntimeMetrics.

Reply:
It would have been pulled in and evicted 3.5.7 but I've explicitly set it now